### DISCUSSION TITLE
CPU or GPU possibly being throttled when playing games

### DISCUSSION THREAD

**Discussion by Z1XCY** [seerge/g-helper#2513] [Q&A]
Hello!

I'm currently using a ASUS TUF Dash F15 FX516PM. I've been playing certain games like The Finals and OSRS and noticed that every few seconds my frames drops to a certain point, and then back to my regular frames that I get. An example of this would be during The Finals where I would normally get around 70-80 FPS, and after a couple seconds my FPS would drop instantly to around 30-40 FPS. I could tell even when my frames drops by listening to the fans on my laptop that would spin less louder. I've been told that this is possibly a case of my CPU or GPU that's being throttled, but I would like to know if I can tweak it somehow to the point that keeps my frames at a stable number, rather than my frames constantly being dropped. I don't know if it's a good place to ask here, but I'm asking here since I'm using G-Helper to control my laptop.

Thank you in advance!

**Comment by seerge** [comment]
@Z1XCY hello, the easiest way to verify that would be to run some in-game monitor (like afterburner's) or nvidia's own and look at temperatures in game.

CPU temp limit is usually 95-96C. If it's stays below that - you are good.
GPU temp limit is 86-87C

**Comment by Z1XCY** [comment]
Hello @seerge, thank you for your response.

I installed and monitored my GPU mostly with MSI Afterburner as u suggested. I checked the temperatures for my CPU and GPU for a while too but they seem to be doing fine while running the game OSRS. I only noticed my GPU1 core clock and GPU1 BUS usage that have the same pattern going up and down **at the same moment** my frames are going up and down, so I can only believe there is some connection with this. Do you happen to know what the problem could be in that case, or even how to resolve it? If it helps in any way, this problem still remains while switching through Silent/Balanced/Turbo modes on my laptop.

Thank you!

**Reply by seerge** [reply]
Do your temps reach values i mentioned in the post above?

**Reply by Z1XCY** [reply]
Hello,

The temperatures stay below the values you mentioned. I think they average around 80 with most games. The only thing I can still think of is to reinstall the drivers, as I'm still suspecting something might be wrong with the GPU. I'll update if anything changes.
